### PR TITLE
refactor: Alias function for safe_sign_and_autofill_transaction called autofill_and_sign

### PR DESCRIPTION
**Pull request by connorjchen**
## High Level Overview of Change

Added alias function for `safe_sign_and_autofill_transaction` called `sign_and_autofill`

### Context of Change

`safe_sign_and_autofill_transaction` is too long and unnecessary -- potentially deprecate/remove in xrpl-py 2.0

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ The `xrpl-py` library automatically populates the `fee`, `sequence` and `last_le
 
 ```py
 from xrpl.models.transactions import Payment
-from xrpl.transaction import send_reliable_submission, safe_sign_and_autofill_transaction
+from xrpl.transaction import send_reliable_submission, sign_and_autofill
```
Comment (mvadari) [inline README.md:208]: IMO we should recommend that people sign and autofill separately - then, they're not passing in a client in the same method that they're signing.

File: snippets/partial_payment.py [hunk 2]
```diff
@@ -33,7 +30,7 @@
 )
 
 # Sign and autofill, then send transaction to the ledger
-signed_trust_set_tx = safe_sign_and_autofill_transaction(trust_set_tx, wallet2, client)
+signed_trust_set_tx = sign_and_autofill(trust_set_tx, wallet2, client)
```
Comment (mvadari) [inline snippets/partial_payment.py:33]: ditto

File: snippets/paths.py [hunk 3]
```diff
@@ -47,4 +47,4 @@
     paths=paths,
 )
 
-print("signed: ", safe_sign_and_autofill_transaction(payment_tx, wallet, client))
```
Comment (mvadari) [inline snippets/paths.py:50]: ditto

File: snippets/reliable_transaction_submission.py [hunk 4]
```diff
@@ -34,7 +31,7 @@
 )
 
 # Sign and autofill the transaction (prepares it to be ready to submit)
-signed_payment_tx = safe_sign_and_autofill_transaction(payment_tx, wallet1, client)
+signed_payment_tx = sign_and_autofill(payment_tx, wallet1, client)
```
Comment (mvadari) [inline snippets/reliable_transaction_submission.py:34]: ditto

**Review by mvadari — COMMENTED** [review]
On second thought, I think it would be better to call this alias `autofill_and_sign`, seeing as that's the order the operations happen in

**Comment by connorjchen** [comment]
Function has been renamed to `autofill_and_sign`

**Comment by mvadari** [comment]
Don't forget to change the PR name :)

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,20 +6,27 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [[Unreleased]]
+
```
Comment (mvadari) [inline CHANGELOG.md:9]: Is there a reason for all of these newlines?

File: README.md [hunk 6]
```diff
@@ -60,26 +57,24 @@ The library supports [Python 3.7](https://www.python.org/downloads/) and later.
 
 [![Supported Versions](https://img.shields.io/pypi/pyversions/xrpl-py.svg)](https://pypi.org/project/xrpl-py)
 
-
```
Comment (mvadari) [inline README.md:63]: ditto - might want to check your VSCode linter settings

**Comment by JST5000** [comment]
Does `safe_sign_and_autofill` still work in the tests? (I.e. is it still exported?)

**Review by intelliot — APPROVED** [review]

**Pull request merged**